pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -496,6 +506,23 @@ func managePod(ctx context.Context, configMapsGetter corev1client.ConfigMapsGett
 	return resourceapply.ApplyConfigMap(configMapsGetter, recorder, configMap)
```

Comment:
Please add a unit to make sure the args order and the configmap content in general is deterministic

Reply:
Will do.